src/main/java/de/chojo/repbot/listener/StateListener.java

```diff
@@ -49,14 +49,10 @@ public void onGuildJoin(@NotNull GuildJoinEvent event) {
                 break;
             }
         }
-
-        guilds.guild(event.getGuild()).migration().migrated();
     }
 
     @Override
     public void onGuildLeave(@NotNull GuildLeaveEvent event) {
-        if (configuration.migration().isActive()) return;
-        // Normally we want to delete all data of a guild after the bot left.
```

Comment:
The comment does not have to much value, but do we want to keep it for consistency with the listeners below?

Reply:
This was mainly added for the verification process.

We could change it to:
> We want to delete all data of a guild after the bot left.
Since there is no other option anymore